chia/data_layer/data_store.py

```diff
@@ -108,29 +109,68 @@ async def get_row_by_index(self, index: int) -> CLVMObject:
     async def get_row_by_hash(self, row_hash: bytes32) -> CLVMObject:
         pass
 
-    async def insert_row(self, table_id: bytes32, clvm_object: CLVMObject) -> None:
+    async def insert_row(self, table: bytes32, clvm_object: CLVMObject, index: Optional[int] = None) -> None:
```

Comment:
does the user determine the row index  on submission ? 
i think that it should just be a running id

Reply:
All the examples had `Row #1` etc so I added indexes.  As I look back, I may well have been reading too much into that and it may be removed.  It is a point to discuss as to whether the rows have order to them.